chirp/drivers/th_uv88.py

```diff
@@ -1094,6 +1261,496 @@ def get_settings(self):
+            valname = ""
+            for i in range(valuelen):
+                if (i // 2) < len(value):
+                    if (i % 2) == 0:
+                        digit = int(value[i // 2]) // 16
+                    else:
+                        digit = int(value[i // 2]) % 16
+                else:
+                    digit = int(getattr(obj, atrb+"last"))
+
+                # Convert
+                valname += OPTSIGTELDIGIT_LIST[digit]
+
+            rx = RadioSettingValueString(0, maxlen, valname, False,
+                                         CHARSET_TEL)
+            rset = RadioSetting(name, display, rx)
+            rset.set_apply_callback(_set_hexlen, obj, atrb, maxlen)
+            setting.append(rset)
+
+        def _add_extdigit(setting, name, display, value, addoff):
+            digits = OPTSIGEXTDIGIT_LIST.copy()
+            if addoff:
+                digits.insert(0, "OFF")
+            digit = digits[value]
+            rx = RadioSettingValueList(digits, digit)
+            rset = RadioSetting(name, display, rx)
+            setting.append(rset)
+
+        def _add_ani(setting, name, display, value):
+            ani = OPTSIGENCTYPE_LIST[value]
+            rx = RadioSettingValueList(OPTSIGENCTYPE_LIST, ani)
+            rset = RadioSetting(name, display, rx)
+            setting.append(rset)
+
+        def _add_time(setting, name, display, start, step, count,
+                      format, value):
+            options = []
+            for i in range(count):
+                options.append(format % (start+i*step))
+            rx = RadioSettingValueList(options, options[value])
+            rset = RadioSetting(name, display, rx)
+            setting.append(rset)
+
+        def _set_int10(setting, obj, atrb):
+            vx = float(str(setting.value))
+            vx = int(vx * 10)
+            setattr(obj, atrb, vx)
+            return
+
+        def _add_int10(setting, name, display, obj, atrb, mini, maxi, value):
+            rx = RadioSettingValueFloat(mini, maxi, value / 10, 0.1, 1)
+            rset = RadioSetting(name, display, rx)
+            # This callback uses the array index
+            rset.set_apply_callback(_set_int10, obj, atrb)
+            setting.append(rset)
+
+        def _add_ttones_tone(setting, name, display, obj, atrb, value):
+            _add_int10(setting, name, display, obj, atrb, 300.0, 3116.0, value)
+
+        def _add_intname(setting, name, display, src, atrb, value):
+            options = []
+            for i in range(len(src)):
+                asname = str(getattr(src[i], atrb))
+                options.append("%02d - " % (i+1) + asname)
+
+            rx = RadioSettingValueList(options, options[value])
+            rset = RadioSetting(name, display, rx)
+            setting.append(rset)
+
+        def _set_mask_array(setting, src, idx):
+            if bool(setting.value):     # Enabled = 1
+                vx = 1
+            else:
+                vx = 0
```

Comment:
I think you should be able to just `int(setting.value)` or just use `setting.value` without the cast, no? Not sure if you did this out of habit or need, but let me know and I could maybe make some more native fixes so they behave properly.

Reply:
Sure, but int(bool) in python will give you a 0 or 1 and save you three lines. Anyway, I know it's copy/paste, just thought I'd point it out since you asked about pythonic code style :)